app/src/interfaces/select-icon/select-icon.vue

```diff
@@ -28,7 +28,10 @@
 		</template>
 
 		<div class="content" :class="width">
-			<template v-for="(group, index) in filteredIcons" :key="group.name">
+			<template v-for="group in filteredIcons" :key="group.name">
+				<v-divider v-if="group.icons.length > 0">
+					{{ group.name }}
+				</v-divider>
```

Comment:
Lets omit the name here to keep things the same as before. If we wanna show the group names, they have to be localized